client/src/components/landing/components/testimonials.tsx

```diff
@@ -11,121 +12,123 @@ const Testimonials = (): JSX.Element => {
   const { t } = useTranslation();
 
   return (
-    <div className='testimonials'>
-      <h2
-        className='big-heading text-center'
-        data-playwright-test-label='testimonials-section-header'
-      >
-        {t('landing.testimonials.heading')}
-      </h2>
-      <div className='testimonials-row' data-test-label='testimonial-cards'>
-        <div
-          className='testimonial-card'
-          data-playwright-test-label='testimonial-card'
+    <Col md={8} mdOffset={2} sm={10} smOffset={1} xs={12}>
```

Comment:
This is not necessarily introducing this issue, but it is enhancing it...

<details><summary>image</summary>

<img width="2168" alt="Screenshot 2024-05-30 at 4 09 49 PM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/20648924/ad39fd64-e1ae-4785-b121-3f6f5ddbee35">

</details>

In the image, production is on the right, this PR is on the left. It adds a bunch of blue/empty space below the image. This issue is already present because there's a window width where you can see a tiny bit of that below the image - but this makes it so there's a large amount. I don't think we want that. Perhaps that extends beyond the scope of this PR.